docs/install.md

```diff
@@ -123,6 +124,104 @@ You can also build Nix from source by following the instructions at https://nixo
 
 See [usage.md](usage.md) for usage instructions, such as how to update.
 
+Tutorial: install and configure NixOS for nix-bitcoin on VirtualBox (macOS host)
+---
+## 1. VirtualBox installation
+The following steps are meant to be run on the machine you deploy from, not the machine you deploy to.
+
+1. Download and install VirtualBox
+	Use the official Downloads page: [https://www.virtualbox.org/wiki/Downloads](https://www.virtualbox.org/wiki/Downloads)
+
+	Make sure that the installer you download shows a lock icon in the top right corner to make sure it is signed by Oracle.
+
+2. Create Host Adapter in VirtualBox
+
+	```
+	vboxmanage hostonlyif create
+	```
+
+## 2. Nix installation
+The following steps are meant to be run on the machine you deploy from, not the machine you deploy to. You can also build Nix from source by following the instructions at https://nixos.org/nix/manual/#ch-installing-source.
+
+1. Install latest Nix in "multi-user mode" with GPG Verification
+
+	```
+	curl -o install-nix https://nixos.org/nix/install
+	curl -o install-nix.sig https://nixos.org/nix/install.sig
+	gpg2 --recv-keys B541D55301270E0BCF15CA5D8170B4726D7198DE
+	gpg2 --verify ./install-nix.sig
+	sh ./install-nix --daemon
+	```
+
+	Then follow the instructions. Open a new terminal window when you're done. Make sure that `nix-shell -p nix-info --run "nix-info -m"` executes successfully. 
+
+2. Optional: Disallow substitutes
+
+	You can put `substitute = false` to your `nix.conf` usually found in `/etc/nix/` to build the packages from source.
+	This eliminates an attack vector where nix's build server or binary cache is compromised.
+
+## 3. LinuxKit Nix installation
+In order to build binaries for your linux (NixOS) virtual machine on a macOS host machine, you need to use [linuxkit-nix](https://github.com/nix-community/linuxkit-nix). It uses hyperkit to spin up a separate VM on which it builds binaries. An alternative solution is [nix-docker-build-slave](https://github.com/LnL7/nix-docker/blob/master/start-docker-nix-build-slave).
+
+1. Installation
+
+	```
+	nix-env -i /nix/store/jgq3savsyyrpsxvjlrz41nx09z7r0lch-linuxkit-builder
+    nix-linuxkit-configure
+	```
+
+	You may want to use `nix-linuxkit-configure -c 4` to give the builder 4 CPUs.
+
+2. Confirm that nix-linuxkit works
+
+    ```
+	nix-build /Users/lev/.cache/nix-linuxkit-builder/example.nix
+	```
+
+	As the installer says, run a `nix-build` to make sure that you are able to build linux binaries. The `example.nix` is specifically configured to force a x86_64-linux build. Remove the generated `result` folder afterwards.
+
+## 4. Nixops deployment
```

Comment:
In general we should avoid repeating the same instructions in this document if possible. How about just referring to the Nixops deployment section in the other tutorial (with the exception of running deploy with the `--option system x86_64-linux` flag)?